lib/views/item/item-detail.js

```diff
@@ -134,7 +181,7 @@ class ItemDetail extends React.Component {
         <Image
           style={styles.distanceImage}
           source={require('../../images/item-scene-room.png')}/>
```

Comment:
TBH I can't quite remember. But don't focus too much on this image as it's being removed in the [map patch](https://github.com/mozilla-magnet/magnet-client/pull/340).